### ISSUE TITLE
Bug: Language not saving

### ISSUE THREAD

**Issue by linsyking** [mattermost/focalboard#2001]
## Steps to reproduce the behavior

I change the language, but when I start the program again, it returns to English.

## Expected behavior

The language should be saved.

## Edition and Platform

 - Edition: Personal Desktop
 - Version: Latest
 - OS: Linux

**Comment by sbishel** [comment]
@ogi-m Can you try to duplicate.

**Comment by ogi-m** [comment]
@sbishel Yup, I could repro on Windows. No matter what the user sets, It defaults to system default on next start up

**Comment by chenilim** [comment]
This might have to do with the app cycling the port. Putting it up for grabs.

**Comment by fegon-syns** [comment]
Along with language, the selected theme and font size are also discarded when starting the program again. I suppose the cause is the same.

**Comment by 1-1is0** [comment]
i have the same issue with themes in v7.4.4

**Comment by 75re** [comment]
> i have the same issue with themes in v7.4.4

same with 7.5.1
